engine/context/ecs-integration.md

```diff
@@ -49,7 +51,7 @@ myecscontext *
 default  Current DOCKER_HOST based configuration   unix:///var/run/docker.sock     swarm

```

Comment:
Does the region not populate the description here?

Reply:
it does not actually for ECS. Also, since the context references AWS profile or env variables, region might change if you chenge the AWS profile definition or change env variables, so we can't easily use region in the description that is stored (duplicated) in the context

Reply:
creating a context from env vars, the description is actually set to "credentials read from environment", should we set this as example ?

Reply:
Yeah I think so :)